frontend/projects/upgrade/src/app/core/feature-flags/store/feature-flags.reducer.ts

```diff
@@ -13,6 +13,7 @@ export const initialState: FeatureFlagState = adapter.getInitialState({
   isLoadingUpdateFeatureFlagStatus: false,
   isLoadingFeatureFlagDetail: false,
   isLoadingFeatureFlagDelete: false,
+  isLoadingSelectedFeatureFlag: false,
```

Comment:
you created this selector but you aren't setting or updating its value anywhere. Are you planing to address that later?

Reply:
oops not all changes were staged, i pushed the right commits, good eye 👀